java/core/src/main/java/co/worklytics/psoxy/Sanitizer.java

```diff
@@ -29,9 +31,9 @@ class Options implements Serializable {
         String defaultScopeId;
 
         //q: add regexes to whitelist endpoints that we actually use??
-        Rules rules;
-
+        Rules1 rules;
 
+        Rules2 rules2;
```

Comment:
backwards compatibility; support for `Rules1` should be maintained, while adding more intuitive `Rules2` with support for more transforms